### PR TITLE
Bump version to 5.2.2

### PR DESCRIPTION
**Pull request by nehaljwani**

### CODE REVIEW

**Comment by jakirkham** [comment]
Thanks @nehaljwani. This feedstock needs an updated token. So will need to go back through `staged-recipes`. More details in this [comment]( https://github.com/conda-forge/conda-forge.github.io/issues/426#issuecomment-321897876 ). Would you have time to do that?

**Comment by nehaljwani** [comment]
@jakirkham https://github.com/conda-forge/staged-recipes/pull/3565

**Comment by jakirkham** [comment]
Seems there is a merge conflict.

**Comment by nehaljwani** [comment]
Conflicts resolved!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -31,15 +31,30 @@ test:
 
   imports:
     - psutil
+    - psutil._psutil_posix    # [unix]
+    - psutil._psutil_linux    # [linux]
+    - psutil._psutil_osx      # [osx]
+    - psutil._psutil_windows  # [win]
```
Comment (jakirkham) [inline recipe/meta.yaml]: We should be careful about testing private modules. Not a problem to test them. However if things change internally, we may need to drop them.

Comment (nehaljwani) [inline recipe/meta.yaml]: Sure, if they change internally, I'll remove/replace them.

**Comment by jakirkham** [comment]
Thanks @nehaljwani.

**Pull request merged**